src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Assistant/TicketAssistantTile.tsx

```diff
@@ -43,6 +43,10 @@ export const TicketAssistantTile: React.FC<TicketAssistantProps> = ({
     (config) => config.configuration.requiresLogin,
   );
 
+  const title = t(TicketingTexts.ticketAssistantTile.title);
+  const description = t(TicketingTexts.ticketAssistantTile.description);
+  const accessibilityLabel = [title, 'Beta', description].join('. ');
```

Comment:
There is somerhing called `screenReaderPause` from `import {screenReaderPause} from '@atb/components/text';` then `.join(screenReaderPause);`